at_secondary/at_secondary_server/lib/src/verb/metrics/metrics_impl.dart

```diff
@@ -275,3 +276,59 @@ class LastPkamMetricImpl implements MetricProvider {
     return 'LastPkam';
   }
 }
+
+class CommitLogCompactionStats implements MetricProvider {
+  static final CommitLogCompactionStats _singleton =
+      CommitLogCompactionStats._internal();
+
+  CommitLogCompactionStats._internal();
+
+  factory CommitLogCompactionStats.getInstance() {
+    return _singleton;
+  }
+
+  @override
+  getMetrics({String? regex}) async {
+    var keyStore = SecondaryPersistenceStoreFactory.getInstance()
+        .getSecondaryPersistenceStore(
+            AtSecondaryServerImpl.getInstance().currentAtSign)!
+        .getSecondaryKeyStore();
+    var atData = await keyStore!.get(commitLogCompactionKey);
+    if (atData != null && atData.data != null) {
+      return atData.data;
+    }
+  }
+
+  @override
+  String getName() {
+    return 'CommitLogCompactionStats';
+  }
+}
+
+class AccessLogCompactionStats implements MetricProvider {
+  static final AccessLogCompactionStats _singleton =
+      AccessLogCompactionStats._internal();
+
+  AccessLogCompactionStats._internal();
+
+  factory AccessLogCompactionStats.getInstance() {
+    return _singleton;
+  }
+
+  @override
+  getMetrics({String? regex}) async {
+    var keyStore = SecondaryPersistenceStoreFactory.getInstance()
+        .getSecondaryPersistenceStore(
+            AtSecondaryServerImpl.getInstance().currentAtSign)!
+        .getSecondaryKeyStore();
+    var atData = await keyStore!.get(accessLogCompactionKey);
+    if (atData != null && atData.data != null) {
```

Comment:
For whatever reason, if the stats are null (atData = null) what should we return. I suggest we return a valid empty JSON.